src/Essentials/src/Share/Share.android.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace Microsoft.Maui.Essentials
 {
-	public static partial class Share
+	public class ShareImplementation : IShare
```

Comment:
@antonfirsov can you please move the implementations to the `Microsoft.Maui.Essentials.Implementations` [namespace](https://github.com/dotnet/maui/blob/main/src/Essentials/src/AppInfo/AppInfo.android.cs#L8-L10)